### PR TITLE
Add parameters to sandbox config

### PR DESCRIPTION
**Pull request by Furisto**
#### What type of PR is this?
/kind feature

#### What this PR does / why we need it:
This adds additional parameters from the run pod sandbox request to the sandbox config

#### Which issue(s) this PR fixes:
None

#### Special notes for your reviewer:
None

#### Does this PR introduce a user-facing change?
```release-note
Add additional parameters from pod sandbox request to sandbox config
```

### CODE REVIEW

File: crates/sandbox/src/lib.rs [hunk 1]
```diff
@@ -91,6 +92,37 @@ pub struct SandboxConfig {
...
+    #[builder(default)]
+    privileged: bool,
+
+    #[get = "pub"]
+    #[builder(default)]
+    seccomp_profile: Option<String>,
+
+    #[get = "pub"]
+    #[builder(default)]
+    readonly_rootfs: bool,
```
Comment (utam0k) [inline crates/sandbox/src/lib.rs:125]: Why don't you extract the security-related stuff with the name LinuxSandboxSecurityContext, etc.? I refered cri-o as a model for this name.

Comment (Furisto) [reply crates/sandbox/src/lib.rs]: Cri-o uses the request as is, whereas in our case sandbox and the CRI service are decoupled, but sure we can group security related stuff together.

**Comment by Furisto** [comment]
@utam0k PTAL

**Review by utam0k — APPROVED** [review]
LGTM

**Pull request merged**